### ISSUE TITLE
[Suggestion] media management for Grav 2.0

### ISSUE THREAD

**Issue by jimblue** [getgrav/grav-plugin-admin#1346]
Hi @rhukster,

I know Grav 2.0 is on his way and you’re looking for nice features to add.
I’ve just found a very a interesting media management system.
It's working with php and js which is nice for Grav though:

https://lychee.electerious.com/

![showcase](https://user-images.githubusercontent.com/10623367/36388064-07b40b1c-159b-11e8-9f55-954ba7ad29c1.jpg)

What do you think about it?

**Comment by rhukster** [comment]
It's nice looking with good functionality but it uses MySQL database so that would not really 'jive' with Grav's functionality.

**Comment by jimblue** [comment]
Yahh indeed, but I though `PHP` and `JS` part of this could interest you.

**Comment by jimblue** [comment]
The frontend is impressive. It's easy to understand without documentation and gives a nice feeling of reactiveness.

Because Media management is IMO the biggest weakness of Grav Admin, that's why I though this open source project could interest you.

It could be nice to add some of the features such as metadata edition or file management (renaming, moving, deleting etc) don't you think?

**Comment by jimblue** [comment]
It's certainly possible to adapte the MySQL part to Grav flatfile system.

**Comment by jimblue** [comment]
Hey @rhukster !

I've been trying `Statamic` which is a flat files CMS using markdown, frontmatter and Laravel under the hood.

It's similar to Grav but `Statamic` also have a very interesting way to manage medias that looks like `Lychee` in simpler.

I'm sure the way it works (backend and frontend) could be inspiring to enhance grav admin media management. It's sharp and clean, nothing superficial.

**Comment by rhukster** [comment]
We’re dedjnitely plannning on better media support so I’m sure it will be a bit of mix of various projects like these.  If you find more leave links here for reference. Cheers.

**Comment by jimblue** [comment]
Sure if I found other interesting libraries or projects about media management I will.
Here is the [Statamic](https://statamic.com/) link BTW!

**Comment by jimblue** [comment]
Hi @rhukster ! I've just found this Statamic video about media management.
This would be much more efficient to handle media like this in Grav don't you think?

https://www.youtube.com/watch?v=FHhlvKEvgPs&list=PLVZTm2PNrzMy932kCVxB_4fSL484d212N&index=3

Cheers

**Comment by jimblue** [comment]
On other suggestion from @paulmassen => #1341

**Comment by Djibdjib** [comment]
Yeah, @jimblue , Statamic is very impressive ! but expensive... I'm agree to you, i would like a medias gallery for Grav.

I'm trying to develop this functionality, come with me :)
